examples/typescript/patch/patch-example.ts

```diff
@@ -1,4 +1,5 @@
-import { CoreV1Api, KubeConfig, setHeaderOptions, JsonPatch } from '@kubernetes/client-node';
+import { CoreV1Api, KubeConfig, setHeaderOptions } from '@kubernetes/client-node';
+import { PatchStrategy } from '@kubernetes/client-node/patch';
```

Comment:
Can we do this without requiring a second import of the client package?

Reply:
yes. updated